tst/standard/isorms.tst

```diff
@@ -566,17 +566,110 @@ true
 gap> ForAll(A, BruteForceInverseCheck);
 true
 
+#T# Errors in checked version of RMSIsoByTriple
+gap> g := SymmetricGroup(4);;
+gap> mat := [[(1, 3), (1, 2)(3, 4)],
+>            [(1, 4, 3, 2), ()],
+>            [(1, 3)(2, 4), (1, 3, 4, 2)]];;
+gap> R := ReesMatrixSemigroup(g, mat);;
+gap> S := ReesMatrixSemigroup(Group((1, 2)), [[()]]);;
+gap> auto := IdentityMapping(g);;
+gap> g_elms_list := [(), (1, 3), (), (), ()];;
+gap> RMSIsoByTriple(R, S, [(), auto, g_elms_list]);
+Error, Semigroups: RMSIsoByTriple:
+<R1> and <R2> are not isomorphic,
+gap> RMSIsoByTriple(R, R, [42, auto, g_elms_list]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[1] should be a permutation,
+gap> RMSIsoByTriple(R, R, [(1, 7), auto, g_elms_list]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[1] should be a permutation on [1 .. 5],
+gap> RMSIsoByTriple(R, R, [(1, 4), auto, g_elms_list]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[1] should not map columns to rows,
+gap> RMSIsoByTriple(R, R, [(), fail, g_elms_list]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[2] should be an isomorphism from
+the underlying group of <R1> to that of <R2>,
+gap> RMSIsoByTriple(R, R, [(), auto, [42, 43, 44]]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[3] should only contain elements from the underlying group of <R2>,
+gap> RMSIsoByTriple(R, R, [(), auto, [(), (), ()]]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[3] should have length equal to
+the number of rows and columns of <R1>,
+gap> RMSIsoByTriple(R, R, [(), auto, g_elms_list]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[3] does not define an isomorphism,
+gap> g_elms_list := [(), (), (), (), ()];;
+gap> RMSIsoByTriple(R, R, [(), auto, g_elms_list]);
+((), IdentityMapping( SymmetricGroup( [ 1 .. 4 ] ) ), [ (), (), (), (), () ])
```

Comment:
Can you add a `BruteForceIsoCheck` for every isomorphism actually created? If they don't take too long that is.